website/content/docs/job-specification/change_script.mdx

```diff
@@ -0,0 +1,84 @@
+---
+layout: docs
+page_title: change_script Stanza - Job Specification
+description: The "change_script" stanza configures a script to be run on template re-render.
+---
+
+# `change_script` Stanza
+
+<Placement groups={['job', 'group', 'task', 'template', 'change_script']} />
+
+The `change_script` stanza allows operators to configure scripts that
+will be executed on template change. This stanza is only used when template
+`change_mode` is set to `script`. 
+
+```hcl
+job "docs" {
+  group "example" {
+    task "server" {
+      template {
+        source      = "local/redis.conf.tpl"
+        destination = "local/redis.conf"
+        change_mode = "script"
+        change_script {
+          command       = "/bin/foo"
+          args          = ["-verbose", "-debug"]
+          timeout       = "5s"
+          fail_on_error = false
+        }
+      }
+    }
+  }
+}
+```
+
+## `change_script` Parameters
+
+- `command` `(string: "")` - Specifies the full path to a script or executable
+  that is to be executed on template change. Path is relative to the driver, e.g.,
```

Comment:
I think it would good to mention this explicitly:

```suggestion
- `command` `(string: "")` - Specifies the full path to a script or executable
  that is to be executed on template change. The command must return exit code 0 to be considered successful. Path is relative to the driver, e.g.,
```